lib/utilities/healthcheckHandler.js

```diff
@@ -3,7 +3,7 @@ const _config = require('../Config').config;
 const data = require('../data/wrapper');
 const vault = require('../auth/vault');
 const metadata = require('../metadata/wrapper');
-const kms = require('../kms/wrapper');
+// const kms = require('../kms/wrapper');
```

Comment:
can you please remove the commented out lines? you can grab it from history!

Reply:
I can remove that one, I prefer to keep `// kms` because it gives some context on the above comment (and IMHO does not hurt in such case, since we want to implement it soon anyway).